Makefile

```diff
@@ -8,4 +8,7 @@ ROOT=$(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 // TODO: add lint and format to github ci
```

Comment:
Also add `format` command and remove `TODO` comment

Reply:
the format command add to the #16 pull request